src/carousel.tsx

```diff
@@ -101,6 +101,15 @@ const arrowButtonStyle = (theme: Theme) => ({
     },
 });
 
+const arrowButtonSeparation = (containerType: ContainerType, isLargueDesktop: boolean) => {
+    switch (containerType) {
+        case 'desktop-wide-column':
+            return isLargueDesktop ? -(24 + arrowButtonSize) : -arrowButtonSize / 2;
+        default:
+            return -arrowButtonSize / 2;
+    }
+};
+
```

Comment:
@yceballost we need your opinion here, as this is not yet defined by design rules, this can change depending on your opinion (see screenshots attached).